lib/auth/permissions.go

```diff
@@ -519,10 +526,12 @@ func GetCheckerForBuiltinRole(clusterName string, clusterConfig services.Cluster
 					MaxSessionTTL: services.MaxDuration(),
 				},
 				Allow: services.RoleConditions{
-					Namespaces:    []string{services.Wildcard},
-					Logins:        []string{},
-					NodeLabels:    services.Labels{services.Wildcard: []string{services.Wildcard}},
-					ClusterLabels: services.Labels{services.Wildcard: []string{services.Wildcard}},
+					Namespaces:     []string{services.Wildcard},
```

Comment:
Are we sure that all those builtins really need a wildcard? What would break if we not add those lines?

Reply:
Many of the tests break without these, I only added the labels which were necessary to fix the tests. I'm not really adding any permissions here, just making more explicit what we are already relying on from the current defaults.

I don't have a deep understanding of these built-in roles yet, so it could just a problem with the tests using these built-in roles incorrectly rather than these permissions being fundamentally necessary. But it seems logical that the db role would need database access, the proxy role would need access to everything, etc. If I'm missing something, we may need to discuss this further.